.changeset/weak-zebras-marry.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Since this is a user-facing change and not just dev facing, I think it should maybe be a patch?